subsys/pcd/src/pcd.c

```diff
@@ -49,13 +40,6 @@ K_TIMER_DEFINE(network_core_finished_check_timer,
 
 #endif /* CONFIG_PCD_APP */
 
-struct pcd_cmd {
```

Comment:
Why is this exposed in the header file now?

Reply:
The struct is needed in header, so that TF-M is able to set the magic. TF-M cannot include this file as it includes Zephyr specific code. The common header does not.